pages/signin/index.tsx

```diff
@@ -43,9 +43,10 @@ const SignInPage = ({ t }: PageProps): React.ReactElement => {
     }
   };
 
+  const errorComponent = error ? <Error error={error} /> : null;
```

Comment:
see above. I think we should not create the errorComponent here and pass it to layout, but pass the message and import the component into layout.